lnclient/ldk/ldk_logger.go

```diff
@@ -1,13 +1,22 @@
 package ldk
 
 import (
-	// "github.com/getAlby/hub/ldk_node"
-	"github.com/getAlby/hub/logger"
+	"os"
+	"path/filepath"
+	"strconv"
 
+	"github.com/getAlby/hub/logger"
 	"github.com/getAlby/ldk-node-go/ldk_node"
+	"github.com/orandin/lumberjackrus"
 	"github.com/sirupsen/logrus"
 )
 
+const ldkLogFilename = "ldk.log"
+const logDir = "log"
+
+var LDKLogger *logrus.Logger
```

Comment:
I don't think we need a singleton logger - it can be as part of the `ldkLogger` struct

Reply:
and ideally closed when we stop the node